log/log.go

```diff
@@ -143,13 +144,19 @@ func (m *defaultLogger) PrintBitriseStartedEvent(plan models.WorkflowRunPlan) {
 			EventType: "bitrise_started",
 		})
 	} else {
-		m.PrintBitriseASCIIArt(plan.Version)
-		m.Warnf("CI mode: %v", plan.CIMode)
-		m.Warnf("PR mode: %v", plan.PRMode)
-		m.Warnf("Debug mode: %v", plan.DebugMode)
-		m.Warnf("Secret filtering mode: %v", plan.SecretFilteringMode)
-		m.Warnf("Secret Envs filtering mode: %v", plan.SecretEnvsFilteringMode)
-		m.Warnf("No output timeout mode: %v", plan.NoOutputTimeoutMode)
+		m.Print("")
+		m.Printf("Invocation started at %s", colorstring.Cyan(m.opts.TimeProvider().Format(consoleTimeLayout)))
+		m.Printf("Bitrise CLI version: %s", colorstring.Cyan(plan.Version))
+		m.Print("")
```

Comment:
nit: you can save some time by omitting the `""` parameter to print a newline. It will work with an empty argument because the logger will append a `\n` char to whatever you supply here. And supplying nothing will end up with juts the single `\n` char.

Reply:
Nice, TIL